.github/pull_request_template.md

```diff
@@ -69,12 +69,12 @@ This section lists items that need to be checked or updated when making changes
 
 When merging a layout, you must ensure that the content type has been turned on for `next-build` inside the `CMS`. This CMS flag must be turned on for editors to preview their work using the next build preview server.
 
-Resource types (layouts) that have not been approved by design should NOT be pushed to production. Ensure that [slug.tsx](../src/pages/[[...slug]].tsx) does not include your resource type if it is not approved.
+Resource types (layouts) that have not been approved by design should NOT be pushed to production. Ensure that [slug.tsx](../src/pages/[[...slug]].tsx) does not include your resource type if it is not approved. If the layout is to be evaluated by QA the resource type and component should be included in the slug.tsx, so that it can reviewed on a tugboat when the .env.tugboat environment variable is set to true.
 
-The status of layouts should be kept up to date inside [templates.md](./templates.md). This includes QA progress, development progress, etc. A link should be provided for where testing can occur.
+The status of layouts should be kept up to date inside [templates.md](../READMEs/templates.md). This includes QA progress, development progress, etc. A link should be provided for where testing can occur.
 
 ## Merging a Non-Approved Layout
 
 Your new resource type should not be included inside [slug.tsx](../src/pages/[[...slug]].tsx). Items added here will go into production once merged into the `main` branch. It is imperative that we do not push anything live that has not been approved.
 
-Ensure that this layout has been added to the [templates.md](./templates.md) file with the current status of the work.
+Ensure that this layout has been added to the [templates.md](../READMEs/templates.md) file with the current status of the work.
```

Comment:
How do you feel about deleting this whole bit? We've decoupled releases from merges into `main` by the use of CMS feature toggles / environment variables. The whole "non-approved" layout doesn't mean anything to any of us anymore, if I'm not mistaken.

cc @timcosgrove for a sanity check.

Reply:
Per [this slack conversation](https://dsva.slack.com/archives/C01SR56755H/p1743016372605289) Tim has no objections to us removing this entirely, go forth and remove!

Reply:
I'll remove

Reply:
Ok @cvalarida I rewrote the section in the pull request template to remove the notion of "approved"/"unapproved" templates and reinforced the needs for the PR to have the resource type in the catchall `[[...slug]].tsx` and the tugboat .env updated.